src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -3864,5 +3863,86 @@ public void addServiceToRequestedInteface(String serviceName) {
   public Map<String, Set<String>> publishInterfaceToPossibleServices() {
     return interfaceToPossibleServices;
   }
+  
+  /**
+   * Saves a "sane" set of embedded defaults constructed for this service
+   * @param className - the class whos defaults will be saved
+   * @return - returns the set of configuration sets successfully saved
+   */
+  static public Set<String> saveDefaults(String className) {
+    try {
+      return saveDefaults(className, null, null);

```

Comment:
you should not pass null for overwrite here.  be explicit.

Reply:
same for the configPrefixPath .. be explicit so it's easier to read the code.. o/w you have to dig through to see that it gets defaulted in the other saveDefaults method.